python/doc/examples/probabilistic_modeling/distributions/plot_conditional_random_vector.py

```diff
@@ -1,31 +1,55 @@
 """
-Create a conditional random vector
-==================================
+Create a deconditioned random vector
+====================================
 """
 
 # %%
-# In this example we are going to build a conditional random vector
+# In this example we are going to build the random vector :math:`\inputRV` defined by the
+# conditional distribution of:
 #
 # .. math::
-#    \vect X|\vect\Theta
 #
+#    \inputRV|\vect{\Theta}
+#
+# where :math:`\vect{\Theta}` is the output of the random variable :math:`\vect{Y}` through
+# the link function :math:`f`:
+#
+# .. math::
+#
+#    \vect{\Theta} & = f(\vect{Y})\\
+#    \vect{Y} & \sim \cL_{\vect{Y}}
+#
+# This example creates a :class:`~openturns.RandomVector` which can only be sampled.
+#
+# We consider the case where :math:`X` is of dimension 1 and follows a exponential distribution
+# defined by:
+#
+# .. math::
+#
+#    X|(\Lambda, \Gamma) & \sim \cE(\Lambda, \Gamma) \\
```

Comment:
There is no universal notation for each distribution: I suggest to explain what means the caligraphic E.